tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,11 +33,10 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
+//For ixgbe and vmxnet3 drivers
```

Comment:
I would prefer to have the remaining changes as a separate commit. Also please use a consistent style for commits summary as the rest of the commits, e.g.:

lkl tools: virtio net dpdk: add max_packet_sz and mbuf_num parmeters